types/fabric/fabric-tests.ts

```diff
@@ -1,3 +1,5 @@
+import { fabric } from 'fabric';
```

Comment:
The sample page https://github.com/kangax/fabric.js clearly demonstrates using this as a global!

Reply:
But in TypeScript we should not use global objects. We should import it and after that use. In case with fabricjs you will import 

```
import { fabric } from 'fabric';
```

And work like with a global object.

```
...
const canvas = new fabric.Canvas(...);
...
```

Anyway, after get yours comment, I was try change type definitions to be able use fabric as global object (like in the sample page), but I could not do this. I was use type definitions of `angularjs` as example and the type definitions of other libraries as well, but this approaches does not work, because merging of namespace does not work becuase the `fabric` namesapce contains classes.

If I just declare the `fabric` namespace and after that declare `export as namespace` statement

```
declare namespace fabric {
...
}

export as namespace fabric;
export = fabric;
```

in tests `fabric` is available as global, but this es6 import does not work

```
import { fabric } from 'fabric';
```

So, if you has some idea how to do it, I will grateful for any suggestions.

Reply:
> But in TypeScript we should not use global objects.
You should generally do what you do in JS. If the library isn't actually a module you shouldn't be trying to import from it.